frontend/src/components/SideNav.tsx

```diff
@@ -0,0 +1,429 @@
+/*
```

Comment:
Does metadata UI need this component?

Reply:
It's used in the router component, and I don't really know how to remove it from there

Reply:
So in metadata UI, router is not rendered, is that right? Then it's fine

Reply:
That's correct!